docs/user-guide/advanced.md

```diff
@@ -4,6 +4,29 @@ title: User Guide (advanced)
 sidebar_label: Advanced options
 ---
 
-Welcome to the user guide!
+There are some options to tweak the invitation link to unlock more features in
+Jitsi. The following parameters apply to the web version only.
```

Comment:
They work on mobile too!

Reply:
Oh, I wasn't aware of that. Let me update the PR…

Reply:
Done.